Lib/ufo2ft/filters/explodeColorLayerGlyphs.py

```diff
@@ -0,0 +1,51 @@
+from ufo2ft.filters import BaseFilter
+from ufo2ft.util import _GlyphSet
+
+
+COLOR_LAYER_MAPPING_KEY = "com.github.googlei18n.ufo2ft.colorLayerMapping"
+COLOR_LAYERS_KEY = "com.github.googlei18n.ufo2ft.colorLayers"
+
+
+class ExplodeColorLayerGlyphsFilter(BaseFilter):
+
+    def start(self):
+        self._layerGlyphSets = {}
+
+    def set_context(self, font, glyphSet):
+        self.globalColorLayerMapping = font.lib.get(COLOR_LAYER_MAPPING_KEY)
+        font.lib[COLOR_LAYERS_KEY] = {}
+        return super().set_context(font, glyphSet)
+        
+    def _getLayer(self, font, layerName):
+        layer = self._layerGlyphSets.get(layerName)
+        if layer is None:
+            layer = _GlyphSet.from_layer(font, layerName)
+            self._layerGlyphSets[layerName] = layer
```

Comment:
same for this `self._layerGlyphSets` cache. It doesn't belong to the Filter object, but to its `self.context` (as defined in `set_context` method) because it is specific to the font currently being filtered.
A filter object can be instantiated once with some parameters, then called on multiple glyphSets from different fonts.

Reply:
Done.